modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/freelist/AbstractFreeList.java

```diff
@@ -267,6 +270,136 @@ private int addRowFragment(
         }
     }
 
+    /** Page handler that is used to update the expiration time of CacheDataRow. */
+    private final PageHandler<T, Boolean> updateDataRowTtl = new UpdateRowMetaHandler();
```

Comment:
Could be static

Reply:
I would prefer to exclude this optional task from the original issue. It could be optimized later, under a separate ticket.